src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
 // write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+function createNotification(type, textMessage) {
+  const notification = document.createElement('div');
+
+  notification.className = type;
+  notification.textContent = textMessage;
+  body.append(notification);
+}
+
+logo.addEventListener('click', () => {
+  const promise1 = new Promise((resolve, reject) => {
+    resolve();
+  });
+
```

Comment:
```suggestion
const promise1 = new Promise((resolve, reject) => {
   logo.addEventListener('click', () => {
    resolve();
  }
});
```
should be listener inside the promise